webserver/jersey/src/test/java/io/helidon/webserver/jersey/JerseyExampleResource.java

```diff
@@ -212,4 +216,22 @@ public Response deleteNotFound(@Context UriInfo uriInfo) {
     	throw new WebApplicationException(Response.status(404).entity("Not Found").build());
     }
 
+    @GET
+    @Path("/streamingOutput")
+    @Produces("application/stream+json")
+    public StreamingOutput getHelloOutputStream() {
+        return out -> {
+                try {
+                    out.write(("{ value: \"first\" }\n").getBytes(StandardCharsets.UTF_8));
+                    out.flush();
+
+                    Thread.sleep(2000);     // wait before sending next chunk
```

Comment:
Minor nit: I know what you're doing here, but it will slow down the CI/CD builds.

Reply:
I'll reduce the wait time, I don't think it needs to be that long.